tests/installation/partitioning/no_separate_home.pm

```diff
@@ -15,10 +15,11 @@
 use parent 'y2_installbase';
 use strict;
 use warnings FATAL => 'all';
+use testapi;
 
 sub run {
     my $partitioner = $testapi::distri->get_partitioner();
-    $partitioner->edit_proposal(has_separate_home => 0);
+    $partitioner->edit_proposal(has_separate_home => 0, multiple_disks => !!get_var('NUMDISKS'));
```

Comment:
Alright, so the problem is that in https://openqa.suse.de/tests/5323544 `NUMDISKS` is set to 1, which is correct. So we cannot use undef value here. So it should be something like 
```
my $multiple_disks = get_var('NUMDISKS', 1)  > 1 ? 1 : 0;
$partitioner->edit_proposal(has_separate_home => 0, multiple_disks => $multiple_disks);
```

Reply:
Thanks for your suggestion, the codes are updated.
The verification is GREEN now: https://openqa.suse.de/tests/5326552#step/no_separate_home/5